examples/dockerdev/docker-compose.yml

```diff
@@ -8,7 +8,6 @@ x-app: &app
       RUBY_VERSION: '2.6.3'
       PG_MAJOR: '13'
       NODE_MAJOR: '14'
-      YARN_VERSION: '1.22.15'
```

Comment:
I think, we should be able to preserve the version; @michsch could you please update the Dockerfile to install not the latest but the specified one?

Reply:
That will look something like: `RUN npm install -g yarn@$YARN_VERSION`

Reply:
Adding Yarn to the sources list can probably go away, as well

Reply:
> That will look something like: RUN npm install -g yarn@$YARN_VERSION

:ok Let's add it (so it would be backward-compatible with the current setup).